packages/logs/lib/client.ts

```diff
@@ -62,7 +62,7 @@ export class LogContext {
         await this.log({ type: 'log', level: 'warn', message, meta, source: 'internal' });
     }
 
-    async error(message: string, meta: (MessageMeta & { error?: unknown }) | null = null): Promise<void> {
+    async error(message: string, meta: (MessageMeta & { error?: unknown; err?: never; e?: never }) | null = null): Promise<void> {
```

Comment:
what's `e`?

Reply:
I just want to prevent us to pass `err` or `e` instead of `error`